config/routes.rb

```diff
@@ -1,3 +1,8 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  resources :users
+  root 'users#new'
+  get '/new', to: 'users#new'
+  get '/all', to: 'users#index'
+  get '/user/:id/edit', to: 'users#edit'
+  post '/users/create', to: 'users#create' 
 end
```

Comment:
[The assignment says to use the `:only` symbol for creating the resources](https://guides.rubyonrails.org/routing.html#restricting-the-routes-created)